souin.go

```diff
@@ -0,0 +1,104 @@
+package httpcache
```

Comment:
Is it possible that we will have other impls under this `httpcache` repo?
If so, release as sub-module is more compact like this repo:https://github.com/hertz-contrib/http3.

Reply:
I think it could be possible, like an implementation into a client that implement the http cache RFC.

`hertz-contrib/httpcache/{implementation}` should be okay, I will rewrite that this evening.

(e.g. `hertz-contrib/httpcache/server`, `hertz-contrib/httpcache/client`)